californium-core/src/main/java/org/eclipse/californium/core/network/UdpMatcher.java

```diff
@@ -295,7 +305,7 @@ public void run() {
 									return;
 								}
 							} catch (Exception ex) {
-								LOGGER.error("error receiving response {} for {}", response, prev, ex);
+								LOGGER.debug("error receiving response {} for {}", response, prev, ex);
```

Comment:
My intention was, that something received, which is broken, floods the log.
But I can go back to error, if you prefer.

Reply:
> My intention was, that something received, which is broken, floods the log.

I get the point, but in the same time, this will be hard to detect unexpected error if we let it in debug. So  "warn" sounds reasonable.